gcpbuildcache/src/main/kotlin/androidx/build/gradle/gcpbuildcache/GcpStorageService.kt

```diff
@@ -80,6 +79,12 @@ internal class GcpStorageService(
         return Companion.delete(storageOptions, blobId)
     }
 
+    override fun validateConfiguration() {
+        if (storageOptions?.service?.get(bucketName, Storage.BucketGetOption.fields()) == null) {
+            throw Exception("Bucket $bucketName under project $projectId cannot be found")
```

Comment:
Oh cool, this error message looks more helpful

Do we know if this error can be thrown if the bucket does exist but isn't accessible?

It looks like the previous error ( https://github.com/googleapis/java-storage/issues/1372#issue-1216467557 ) also printed the role name; I wonder if we need that too

Reply:
I dont know how to get the role name, but I updated to list that it could also be a credentials issue.